app/views/shared/transactions/_table_inside.html.haml

```diff
@@ -19,7 +19,7 @@
       %th.currency= OrderDetail.human_attribute_name("cost")
       %th= OrderDetail.human_attribute_name("order_status")
   %tbody
-    - query_in_slices(order_details).each do |order_detail|
+    - order_details.each do |order_detail|
```

Comment:
This ended up no longer being necessary since the driver has fixed its issues with eager loading association to avoid more than 1000 items.